org.scala-ide.sdt.core/src/org/scalaide/refactoring/internal/rename/Rename.scala

```diff
@@ -49,7 +50,11 @@ class Rename extends RefactoringHandler {
           new refactoring.FileSelection(source.file, tree, start, end)
         }
 
-        selection map refactoring.prepare flatMap (_.right.toOption) map {
+        val preparation = selection flatMap { selection =>
```

Comment:
generally I try to avoid to give same name to option and value but... not sure, semantically it represents same, so ok for me